src/test/resources/testResult/xml/e6a4cc2de5bf437e83efe0107cf026ac_1095.xml

```diff
@@ -31,12 +31,23 @@
       <om:observedProperty xlink:href="https://waterdata.usgs.gov/nwisweb/rdf?parmCd=72019" xlink:title="Depth to water level, ft below land surface"/>
       <om:featureOfInterest xlink:href="https://unittest/api/observations/collections/[collectionId]/items/USGS-07227448?f=json" xlink:title="Punta De Agua Ck ñr Channing, TX"/>
       <om:result>
-        <wml2:point>
-          <wml2:MeasurementTVP>
-            <wml2:time>[NOW-1095DAYS]</wml2:time>
-            <wml2:value>14.1</wml2:value>
-          </wml2:MeasurementTVP>
-        </wml2:point>
+        <wml2:MeasurementTimeseries>
+          <wml2:DefaultTVPMeasurementMetadata>
+            <wml2:uom xlink:title="ft" xlink:href="http://www.opengis.net/def/uom/UCUM/degC"/>
+          </wml2:DefaultTVPMeasurementMetadata>
+          <wml2:point>
+            <wml2:MeasurementTVP>
+              <wml2:time>[NOW-1095DAYS]</wml2:time>
+              <wml2:value>14.1</wml2:value>
+              <wml2:metaData>
+                <wml2:TVPMeasurementMetadata>
+                  <wml2:qualifier>Revised</wml2:qualifier>
+                  <wml2:status>Estimated</wml2:status>
```

Comment:
I know this is a unit test but Revised and Estimated are exclusive qualifiers. Estimated is used for data that is provisional and once approved the estimated is removed. Then if approved data must be adjusted it is labeled as revised.